### PR TITLE
Add Kubernetes setup help markdown

### PR DESCRIPTION
**Pull request by peyao**
Adding some help contents is easier than removing the Help button from the page. So here it is. Feel free to modify my PR directly if you want to change anything.

Screen Shot 2020-07-22 at 3 29 52 AM

### CODE REVIEW

**Comment by peyao** [comment]
Added the word "manual" when referring to "instructions to deploy the required components yourself"

File: cloud_integrations_add_kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Adding a Kubernetes Cluster
+
+Adding a Kubernetes cluster for monitoring requires deploying a **Wavefront proxy** and **Wavefront collector** inside your Kubernetes cluster.
+
+You can follow the step-by-step guided Helm instructions if applicable, or follow the manual instructions to deploy the required components yourself.
```
Comment (rkempf42) [inline cloud_integrations_add_kubernetes.md:5]: You have two choices: 
* Follow the simple Helm install steps on the left. 
* Perform manual installation ( ??link??).

File: cloud_integrations_add_kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+If you need more help, check out the [Kubernetes Troubleshooting](https://docs.wavefront.com/wf_kubernetes_troubleshooting.html) page.
```
Comment (rkempf42) [inline cloud_integrations_add_kubernetes.md:7]: A lot of the Help panels follow the pattern: 

`### More Info`
<links> 
And, you could link to the Troubleshooting page and to the SDK README.

**Comment by peyao** [comment]
Pushed changes according to your advice.

**Pull request merged**